ethereumetl/streaming/postgres_tables.py

```diff
@@ -127,4 +128,22 @@
     Column('trace_id', String, primary_key=True),
 )
 
+TOKENS = Table(
+    'tokens', metadata,
+    Column('address', VARCHAR(42), primary_key=True),
+    Column('name', String),
+    Column('symbol', String),
+    Column('decimals', Integer),
+    Column('function_sighashes', ARRAY(String)),
+    Column('total_supply', Numeric(78)),
+)
 
+CONTRACTS = Table(
+    'contracts', metadata,
```

Comment:
Since contracts can be redeployed with CREATE2 opcode, address can't be the primary key. The same goes for the tokens table. Would it be possible to have a composite key here (address, block_number)?

Reply:
Good point and thank you for the feedback! Yes, this is possible and is added in https://github.com/blockchain-etl/ethereum-etl/pull/293/commits/1a0bac2e2c14a160c45ee4aecc3af12176caf63a.